exchanges/btse/btse.go

```diff
@@ -55,14 +57,33 @@ const (
 	btseCancelAllAfter   = "order/cancelAllAfter"
 )
 
+// FetchFundingHistory gets funding history
+func (b *BTSE) FetchFundingHistory(symbol string) (map[string][]FundingHistoryData, error) {
+	var resp map[string][]FundingHistoryData
+	params := url.Values{}
+	if symbol != "" {
+		params.Set("symbol", symbol)
+	}
+	return resp, common.SendHTTPGetRequest(btseAPIURL+btseFuturesPath+btseFuturesFunding+params.Encode(),
+		true,
+		false,
+		&resp)
+}
+
 // GetMarketSummary stores market summary data
 func (b *BTSE) GetMarketSummary(symbol string, spot bool) (MarketSummary, error) {
 	var m MarketSummary
 	path := btseMarketOverview
 	if symbol != "" {
 		path += "?symbol=" + url.QueryEscape(symbol)
 	}
-	return m, b.SendHTTPRequest(http.MethodGet, path, &m, spot, queryFunc)
+	return m, b.SendHTTPRequest(exchange.RestSpot, http.MethodGet, path, &m, spot, queryFunc)
+}
+
+// GetSpotMarkets returns a list of spot markets available on BTSE
+func (b *BTSE) GetSpotMarkets() ([]SpotMarket, error) {
```

Comment:
Add coverage

Reply:
I have checked their api documentation and could not find that endpoint. Have removed the endpoint for now.
https://www.btse.com/apiexplorer/spot/#btse-spot-api-public-endpoints